src/Relogify.Android/MainActivity.fs

```diff
@@ -12,14 +12,63 @@ open Android.Widget
 open Android.OS
 open Xamarin.Forms.Platform.Android
 
+open Android.Util
+open Android.Gms.Common
+
 [<Activity (Label = "Relogify.Android", Icon = "@mipmap/icon", Theme = "@style/MainTheme", MainLauncher = true, ConfigurationChanges = (ConfigChanges.ScreenSize ||| ConfigChanges.Orientation), ScreenOrientation = ScreenOrientation.Portrait)>]
 type MainActivity() =
     inherit FormsAppCompatActivity()
+
+    let TAG: string = "MainActivity"
+    static member CHANNEL_ID: string = "my_notification_channel" // TODO: "Challenges from other players"
+    static member NOTIFICATION_ID: int = 123

```

Comment:
Set notificationId to the HashCode of the message, and save along with the challenges in local storage. This Id can then be used to:
1. Uniquely identify the notification
2. Couple the notification to a saved challenge
3. Remove/cancel the notification when declining the corresponding challenge.